mysite/templates/mysite/change_password.html

```diff
@@ -0,0 +1,11 @@
+{% extends "mysite/base_generic.html" %}
+
+{% block content %}
+
+<form method="post">
+  {% csrf_token %}
+  {{ form }}
+  <button type="submit">Save changes</button>
```

Comment:
text should be Update Password/Reset Password

Reply:
Feedback Incorporated.